dev/tests/integration/testsuite/Magento/Catalog/Model/Product/Attribute/CreateAttributeViaApi.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Magento\Catalog\Model\Product\Attribute;
+
+class CreateAttributeViaApi extends \PHPUnit\Framework\TestCase
```

Comment:
Change name to Repository (you testing \Magento\Catalog\Model\Product\Attribute\Repository behavior, test namespace and name must be the same)